xarray/core/formatting_html.py

```diff
@@ -341,3 +343,130 @@ def dataset_repr(ds) -> str:
     ]
 
     return _obj_repr(ds, header_components, sections)
+
+
+def summarize_datatree_children(children: Mapping[str, Any]) -> str:
```

Comment:
Is this `Any` really a `DataTree`? (Not 100% sure, but it seems like a good guess)

Reply:
Thanks for fixing up the type hints!